src/Aspire.Dashboard/Components/Pages/ConsoleLogs.razor.cs

```diff
@@ -93,17 +89,25 @@ private sealed class ConsoleLogsSubscription
     private readonly List<MenuButtonItem> _resourceMenuItems = new();
 
     // State
+    private bool _hideTimestamp;
     public ConsoleLogsViewModel PageViewModel { get; set; } = null!;
 
     public string BasePath => DashboardUrls.ConsoleLogBasePath;
     public string SessionStorageKey => BrowserStorageKeys.ConsoleLogsPageState;
+    private const string HideTimestampStorageKey = BrowserStorageKeys.ConsoleLogTimestampState;
```

Comment:
I don't think this is nessessary. Use the property on BrowserStorageKeys directly.